docs/_documentations/image-registry-credentials.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+title: Image registry guidance for Eclipse Codewind
+description: Image registry guidance for Eclipse Codewind
+keywords: users, projects, Kubernetes, image, registry, deployment, push
+duration: 5 minutes
+permalink: dockerregistry
+type: document
+parent: installoncloud
+order: 1
+---
+# Codewind Docker registry guidance
+When you run Codewind on Kubernetes, Codewind uses [`buildah`](https://github.com/containers/buildah) to build container images from Dockerfiles and to push the images to an image registry that you specify. You can use the `Image Registry Manager` in your Codewind IDE to configure the registry to use. 
+
+Codewind is flexible with the registry that can be used as long as `buildah` and the node that Codewind is running on can resolve the registry hostname. If `buildah` is unable to resolve the registry hostname,`buildah` is unable to push your projects to the registry, and deployment on Kubernetes is prevented.
+
+Before you proceed, see the [examples](#examples) section on recommended values for common registries:
+
+## Examples:
+The following deployment registries have been tested and verified with Codewind:
+- Dockerhub:
+    - Host: `docker.io`
+    - Codewind deployment registry: `docker.io/<username>`
+    - Credentials: Dockerhub user name and password
+- Quay.io:
+    - Host: `https://quay.io`
+    - Codewind deployment registry: `https://quay.io/<username>`
+    - Credentials: Quay.io user name and encrpyted password
+- Artifactory
+    - Host: `<artifactory-hostname>`
+    - Codewind deployment registry: `<artifactory-hostname>/<username>`
+    - Credentials: Artifactory user name and token
+- OpenShift Registry
+    - Host: `docker-registry.default.svc:5000`
+    - Codewind deployment registry: `docker-registry.default.svc:5000/<project>`
+    - Credentials: OpenShift user name and token
+    
+### Specifying Image Registries for Codewind 
```

Comment:
I might not have mentioned this before...but for titles we only capitalize the first word of the title and any names that are normally capitalized. Doc titles and section titles are not like book titles. ;)